src/Snt/Capi/Repository/Article/FindByChangelogParameters.php

```diff
@@ -47,12 +49,21 @@ public static function createForPublicationIdFromArray($publicationId, array $pa
         $acceptedParameters = [
             'limit',
             'offset',
-            'timeRange',
         ];
         
         $self = new self();
         $self->publicationId = $publicationId;
 
+        if (isset($parameters['since'], $parameters['until'])) {
+            try {
+                $self->timeRange = new TimeRangeParameter(
+                    new DateTime($parameters['since']),
+                    new DateTime($parameters['until'])
+                );
+            } catch (Exception $exception) {
+            }
```

Comment:
do not catch exception here, SDK user should know when `$parameters` are incorrect. IMHO we can leave it just without try..catch clause